scripts/dockerhost/cleanup-dockerhosts

```diff
@@ -14,5 +13,4 @@ do
     echo "Cleaning $hostname"
     sudo docker exec $hostname service docker stop
     sudo docker rm -f $hostname
-    sudo ip link delete $i-ext
```

Comment:
It seems to be automatically deleted when docker container is removed. Calling this was returing an error as the link is not present.